packages/integrations/preact/README.md

```diff
@@ -104,6 +104,23 @@ export default defineConfig({
 
 With the `compat` option enabled, the Preact integration will render React components as well as Preact components in your project and also allow you to import React components inside Preact components. Read more in [“Switching to Preact (from React)”](https://preactjs.com/guide/v10/switching-to-preact) on the Preact website.
 
+When importing React component libraries, in order to swap out the `react` and `react-dom` dependencies as `preact/compat`, you can use [`overrides`](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#overrides) to do so.
+
+```js
+// package.json
+{
+  "overrides": {
+    "react": "npm:@preact/compat@latest",
+    "react-dom": "npm:@preact/compat@latest"
+  }
+}
+```
+
+Check out the [`pnpm` overrides](https://pnpm.io/package_json#pnpmoverrides) and [`yarn` resolutions](https://yarnpkg.com/configuration/manifest#resolutions) docs for their respective overrides features.
+
+> **Note**
+> Currently, the `compat` option only works for React libraries that export code as ESM. If an error happens during build-time, try adding the library to `vite.ssr.noExternal: ['the-react-library']` in your `astro.config.mjs` file.
```

Comment:
not related to the content but i just noticed the github note syntax creates a blockquote 

i'm guessing we don't want to use the custom admonition syntax
```
:::note
Currently, the `compat`...
:::
```

since the github syntax is consistent, maybe there's a transformation that could happen in remark? /cc @sarah11918 @delucis

Reply:
Well spotted @mayank99! That's actually something I've been wanting to add: use GitHub's syntax in these files and then automatically transform them in docs https://github.com/withastro/docs/issues/967